### ISSUE TITLE
Error generating dataset if a slot is the first item in sentence

### ISSUE THREAD

**Issue by Brannonj96** [snipsco/snips-nlu#726]
Title pretty much explains it.  I get the following error if a slot is the first item in an utterance.

```
Traceback (most recent call last):
  File "file\path\python\python36-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "file\path\python\python36-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "file\path\Python\Python36-32\Scripts\snips-nlu.exe\__main__.py", line 9, in <module>
  File "file\path\python\python36-32\lib\site-packages\snips_nlu\__main__.py", line 39, in main
    plac.call(commands[command], sys.argv[1:])
  File "file\path\python\python36-32\lib\site-packages\plac_core.py", line 328, in call
    cmd, result = parser.consume(arglist)
  File "file\path\python\python36-32\lib\site-packages\plac_core.py", line 207, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "file\path\python\python36-32\lib\site-packages\snips_nlu\cli\generate_dataset.py", line 17, in generate_dataset
    dataset = Dataset.from_yaml_files(language, list(files))
  File "file\path\python\python36-32\lib\site-packages\snips_nlu\dataset\dataset.py", line 95, in from_yaml_files
    for doc in yaml.safe_load_all(f):
  File "file\path\python\python36-32\lib\site-packages\yaml\__init__.py", line 84, in load_all
    yield loader.get_data()
  File "file\path\python\python36-32\lib\site-packages\yaml\constructor.py", line 31, in get_data
    return self.construct_document(self.get_node())
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 27, in get_node
    return self.compose_document()
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "file\path\python\python36-32\lib\site-packages\yaml\composer.py", line 110, in compose_sequence_node
    while not self.check_event(SequenceEndEvent):
  File "file\path\python\python36-32\lib\site-packages\yaml\parser.py", line 98, in check_event
    self.current_event = self.state()
  File "file\path\python\python36-32\lib\site-packages\yaml\parser.py", line 393, in parse_block_sequence_entry
    "expected <block end>, but found %r" % token.id, token.start_mark)
yaml.parser.ParserError: while parsing a block collection
  in "initialTraining.yaml", line 34, column 3
expected <block end>, but found '<scalar>'
  in "initialTraining.yaml", line 39, column 10
```

Here is a .yaml snippet that causes it

```
---
type: intent
name: convertUnits
slots:
  - name: fromUnit
    entity: unit
  - name: toUnit
    entity: unit
  - name: num
    entity: snips/number
utterances:
  - [fromUnit](meter) to [toUnit](kilometer)
  - [num](25) [fromUnit](grams) to [toUnit](ounces)
```

Relevant specs: Python 3.6, snips version 0.18.0, windows 10 platform
Edit: Markdown autocorrected the utterances.

**Comment by adrienball** [comment]
Closing as it is a duplicate of https://github.com/snipsco/snips-nlu/issues/719

TLDR: in such cases, you must put the utterances between double quotes`"`
